ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/base-fee-input/base-fee-input.js

```diff
@@ -105,25 +105,57 @@ const BaseFeeInput = () => {
   const [editingInGwei, setEditingInGwei] = useState(false);
 
   const [maxBaseFeeGWEI, setMaxBaseFeeGWEI] = useState(() => {
-    if (
-      estimateUsed !== PRIORITY_LEVELS.CUSTOM &&
-      advancedGasFeeValues?.maxBaseFee
+    if (estimateUsed !== PRIORITY_LEVELS.CUSTOM) {
+      if (advancedGasFeeValues?.maxBaseFeeGWEI) {
+        return advancedGasFeeValues.maxBaseFeeGWEI;
+      } else if (advancedGasFeeValues?.maxBaseFeeMultiplier) {
+        return multiplyCurrencyValues(
+          estimatedBaseFee,
+          advancedGasFeeValues.maxBaseFeeMultiplier,
+          numberOfDecimalsPrimary,
+        );
+      }
+    } else if (
+      advancedGasFeeValues?.maxBaseFeeGWEI &&
+      advancedGasFeeValues?.maxBaseFeeGWEI === maxFeePerGas
+    ) {
+      return advancedGasFeeValues.maxBaseFeeGWEI;
+    } else if (
+      advancedGasFeeValues?.maxBaseFeeMultiplier &&
+      advancedGasFeeValues?.maxBaseFeeMultiplier === maxFeePerGas
     ) {
       return multiplyCurrencyValues(
         estimatedBaseFee,
-        advancedGasFeeValues.maxBaseFee,
+        advancedGasFeeValues.maxBaseFeeMultiplier,
         numberOfDecimalsPrimary,
       );
     }
     return maxFeePerGas;
   });
 
   const [maxBaseFeeMultiplier, setMaxBaseFeeMultiplier] = useState(() => {
-    if (
-      estimateUsed !== PRIORITY_LEVELS.CUSTOM &&
-      advancedGasFeeValues?.maxBaseFee
+    if (estimateUsed !== PRIORITY_LEVELS.CUSTOM) {
+      if (advancedGasFeeValues?.maxBaseFeeMultiplier) {
+        return advancedGasFeeValues.maxBaseFeeMultiplier;
+      } else if (advancedGasFeeValues?.maxBaseFeeGWEI) {
+        return divideCurrencyValues(
+          advancedGasFeeValues.maxBaseFeeGWEI,
+          estimatedBaseFee,
+        );
+      }
+    } else if (
+      advancedGasFeeValues?.maxBaseFeeMultiplier &&
+      advancedGasFeeValues?.maxBaseFeeMultiplier === maxFeePerGas
     ) {
-      return advancedGasFeeValues.maxBaseFee;
+      return advancedGasFeeValues.maxBaseFeeMultiplier;
+    } else if (
+      advancedGasFeeValues?.maxBaseFeeGWEI &&
+      advancedGasFeeValues?.maxBaseFeeGWEI === maxFeePerGas
```

Comment:
Can `maxFeePerGas` now be either a multiplier or GWEI? If so, it seems like that could be very confusing for someone else looking through this. Is it worth it, given these new changes, to now rename this to something like `maxFeePerGasAsGweiOrMultiplier`?

Reply:
`maxFeePerGas` is a property from txParams from transaction in the state, renaming that might impact our transaction flow.